addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -584,15 +576,18 @@ function makeActionManager(env) {
         if (options.lazyController) {
             controllerArray.unshift(options.lazyController);
         }
-        const nextStack = controllerStack.slice(0, index).concat(controllerArray);
+        nextStack.length = 0;
+        nextStack.push(...controllerStack.slice(0, index).concat(controllerArray));
```

Comment:
`nextStack.push(...controllerStack.slice(0, index), controllerArray);`
I guess there is no way to do like in python: array[:index] :)